frontend/src/js/controllers/dashCtrl.js

```diff
@@ -49,12 +52,14 @@
 
         vm.hostChallenge = function() {
 
-            var alert = $mdDialog.alert()
-                .title('Host a challenge')
-                .htmlContent('Please send an email to <a href="mailto:admin@cloudcv.org" class="blue-text">admin@cloudcv.org</a> with the details of the challenge and we will get back soon.')
-                .ok('Close');
-
-            $mdDialog.show(alert);
+            if ($rootScope.isAuth === true)
+            {
+                $state.go('web.challenge-host-teams');
+            }
+            else {
+                utilities.storeData('redirectUrl', 'web.challenge-host-teams');
```

Comment:
Don't use local storage for storing URL's , let me know if there is a specific need.